yang/ietf-l3vpn-ntw.yang

```diff
@@ -2125,6 +2141,84 @@ module ietf-l3vpn-ntw {
                           "Indicates whether IPv4, IPv6, or both
                            address families are to be activated.";
                       }
+                      container timers {
+                        description
+                          "Indicates the RIP timers.";
+                        reference
+                         "RFC 8695: A YANG Data Model for the 
+                         Routing Information Protocol (RIP)";
+                        leaf update-interval {
+                           type uint16 {
+                             range "1..32767";
+                           }
+                           units "seconds";
+                           default "30";
+                          description
+                            "Indicates the RIP update time.
+                             The amount of time for which 
+                             routing updates are sent.";
+                        }
+                        leaf invalid-interval  {
+                          type uint16 {
+                             range "1..32767";
+                           }
+                           units "seconds";
+                           default "180";
+                          description
+                            "Interval before a route is declared 
+                            invalid after no updates are received. 
+                            This value is at least three times
+                            the value for the update-interval argument.";
+                        }
+                        leaf holddown-interval {
+                           type uint16 {
+                             range "1..32767";
+                           }
+                           units "seconds";
+                           default "180";
+                          description
+                            "Interval before better routes are released.";
+                        }
+                        leaf flush-interval {
+                            type uint16 {
+                               range "1..32767";
+                             }
+                             units "seconds";
+                             default "180";
+                          description
+                            "Indicates the RIP  flush timer
+                             The amount of time (in seconds) 
+                             that must pass before a route 
+                             is removed from the routing table.";
+                        }
+                      }
+                      leaf-list neighbor {
+                        type inet:ip-address;
+                        description
+                          "Indicates the IP address of the peer.";
+                      }
+                      uses vpn-common:service-status;
+                    }
+                    container ntp {
```

Comment:
Why is this proposed to be at the routing level? Please consider removing it from the routing container + discuss it as a seprate item. Thanks

Reply:
as a protocol I guess it fits in this container. I have moved out, please see 0af2d397e3e5d818e879360a071957a65012de65

Reply:
Thanks. May be it is better to have it under "services".

Reply:
I guess it would have been good to document first how to use the NTP in the VPN :-) 
Regarding NTP, we need configuration at two levels:

a) At service level (as med suggests): Here you will indicate if you want to active NTP to be activated in the VPN and set common NTP parameters such as authentication (which I see missing by the way)
b) At VPN network access. You need to explicitly enable the broadcast of NTP messages per interface. 

The NTP model is avialable at https://datatracker.ietf.org/doc/draft-ietf-ntp-yang-data-model/?include_text=1 

I guess we need to be (as in the case of BGP) careful on what do we need to expose in the service model about NTP, so that the controller has enough information to activate the NTP... In our templates we don't configure much... but I've seen that the NTP yang is really rich...

Reply:
> a) At service level (as med suggests): Here you will indicate if you want to active NTP to be

I was actually referring to "services" (not service) which is under vpn-network-access.

Reply:
moved as "services" level. Enable the NTP on one VPN NET ACCESS enables NTP at whole VPN Node level. Please see e4f771ad0180e26c66da599923d956639cd33f3b

Reply:
Thanks, Samier. 

Some text is needed to explain the rationale for having it included in the module and why only these leafs are covered.